bootstrap/bootstrap.md

```diff
@@ -7,9 +7,9 @@ About 4 hours 45 minutes
 - 4 hours for Independent Practice & Challenge
 
 ### Prerequisites
-- "HTML" lesson
-- "CSS" lesson
-- "UX/UI" lesson
+- [HTML lesson](/html/html.md)
+- [CSS lesson](/css/css.md)
+- [UX/UI lesson](https://github.com/Techtonica/curriculum/blob/master/ui-ux-design/ui-ux-design.md)
```

Comment:
Hi @yati1998 , it would be better if you change this to "UI/UX lesson" and change this in other files as well.